src/diffusers/pipelines/stable_diffusion_3/pipeline_stable_diffusion_3.py

```diff
@@ -691,17 +734,19 @@ def __call__(
         negative_prompt_embeds: Optional[torch.FloatTensor] = None,
         pooled_prompt_embeds: Optional[torch.FloatTensor] = None,
         negative_pooled_prompt_embeds: Optional[torch.FloatTensor] = None,
+        ip_adapter_image: Optional[PipelineImageInput] = None,
+        ip_adapter_image_embeds: Optional[List[torch.Tensor]] = None,
         output_type: Optional[str] = "pil",
         return_dict: bool = True,
-        joint_attention_kwargs: Optional[Dict[str, Any]] = None,
+        joint_attention_kwargs: Dict[str, Any] = {},
```

Comment:
We may change the default for `joint_attention_kwargs` across all pipelines etc but for now we should keep it consistent